docs/01_getting_ready_for_class.md

```diff
@@ -20,7 +20,7 @@ You can set up your free account by following these steps:
 
 If you already have an account, verify that you can visit github.com within your organization's network.
 
-GitHub is designed to run on the current versions of all major browsers. In particular, if you use Microsoft's Internet Explorer (IE), you must be using the latest version. Take a look at our list of [supported browsers](https://help.github.com/articles/supported-browsers/).
+GitHub is designed to run on the current versions of all major browsers. Take a look at our list of [supported browsers](https://help.github.com/articles/supported-browsers/).
```

Comment:
```suggestion
GitHub is designed to run on the current versions of all major browsers. Take a look at our list of [supported browsers](https://docs.github.com/github/getting-started-with-github/supported-browsers).
```

Reply:
Thanks, @tatarhy! Just for future reference, you can quickly and easily apply suggested changes right here from the pull request with the click of a button; no need to make the changes manually: https://docs.github.com/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

That might save you some time and effort in the future. 😉